src/app/router/navigationDrawer.tsx

```diff
@@ -86,6 +86,7 @@ export const NavigationDrawer = (): JSX.Element => {
             itemFontColor={theme.palette.text.primary}
             divider={false}
             activeItem={activeRoute}
+            activeItemFontColor={theme.palette.primary.main}
```

Comment:
I don't know why but I have to specify this to make the active menu item the theme color. I assume that the itemFontColor overrode it.